homeassistant/components/weatherkit/const.py

```diff
@@ -0,0 +1,13 @@
+"""Constants for WeatherKit."""
+from logging import Logger, getLogger
+
+LOGGER: Logger = getLogger(__package__)
+
+NAME = "Apple WeatherKit"
+DOMAIN = "weatherkit"
+ATTRIBUTION = "Data provided by Apple Weather. https://developer.apple.com/weatherkit/data-source-attribution/"
```

Comment:
Please break long strings around max 88 characters per line. I'd break here before the URL.